src/main/res/layout/seek_bar_preference.xml

```diff
@@ -1,47 +1,75 @@
 <?xml version="1.0" encoding="UTF-8"?>
 
-<RelativeLayout
+<LinearLayout
     xmlns:android="http://schemas.android.com/apk/res/android"
-    android:id="@android:id/widget_frame"
     android:layout_width="match_parent"
     android:layout_height="wrap_content"
-    android:paddingBottom="5dp"
-    android:paddingLeft="15dp"
-    android:paddingRight="10dp"
-    android:paddingTop="5dp" 
+    android:minHeight="?android:attr/listPreferredItemHeight"
+    android:paddingRight="?android:attr/scrollbarSize"
     >
-    <TextView
-        android:id="@+id/seekBarPrefUnitsRight"
-        android:layout_width="wrap_content"
-        android:layout_height="wrap_content"
-        android:layout_alignParentRight="true"
-        android:layout_alignParentTop="true" 
-        />
-    <TextView
-        android:id="@+id/seekBarPrefValue"
-        android:layout_width="wrap_content"
-        android:layout_height="wrap_content"
-        android:layout_toLeftOf="@id/seekBarPrefUnitsRight"
-        android:gravity="right" 
-        />
-    <TextView
-        android:id="@+id/seekBarPrefUnitsLeft"
-        android:layout_width="wrap_content"
-        android:layout_height="wrap_content"
-        android:layout_toLeftOf="@id/seekBarPrefValue"
-        />
-    <LinearLayout
-        android:id="@+id/seekBarPrefBarContainer"
+    <RelativeLayout
         android:layout_width="match_parent"
-        android:layout_height="wrap_content"
-        android:layout_alignParentLeft="true"
-        android:layout_below="@+id/seekBarPrefUnitsRight"
-        android:orientation="horizontal" 
+        android:layout_height="match_parent"
+        android:layout_marginLeft="15dp"
+        android:layout_marginRight="6dp"
+        android:layout_marginTop="6dp"
+        android:layout_marginBottom="6dp"
+        android:gravity="center_vertical"
         >
-        <SeekBar
-            android:id="@+id/seekBarPrefSeekBar"
+        <TextView
+            android:id="@android:id/title"
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            android:layout_alignParentStart="true"
+            android:layout_toStartOf="@+id/seekBarPrefValue"
+            android:singleLine="true"
+            android:textAppearance="?android:attr/textAppearanceLarge"
+            android:textStyle="bold"
+            android:ellipsize="marquee"
+            android:fadingEdge="horizontal"
+            android:layout_alignParentTop="true"
+            />
```

Comment:
Title. Need to figure out what styling is applied to titles by default so we can use it here. Aligned to the edge of the value, so a long title will ellipsize instead of overlapping the value, and to the left, because otherwise a short title will float in the middle.